mintpy/add_attribute.py

```diff
@@ -74,7 +74,7 @@ def update_file_attribute(fname, atr_new):
     return fname
 
 
-def main(argv):
+def main(argv=None):
```

Comment:
Entrypoints get called with no arguments, so this needs to be optional. Seems to work fine with just this change.

Reply:
LGTM.